src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/reporting/ndmis/performance/NdmisAppointmentPerformanceReportProcessor.kt

```diff
@@ -0,0 +1,42 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.reporting.ndmis.performance
+
+import mu.KLogging
+import net.logstash.logback.argument.StructuredArguments.kv
+import org.springframework.batch.item.ItemProcessor
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Referral
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.service.DeliverySessionService
+
+@Component
+class NdmisAppointmentPerformanceReportProcessor(
```

Comment:
tiny nit: i think `NdmisPerformanceReportAppointmentProcessor` etc for these classes is more readable 

you could even just get away with `AppointmentProcessor` - that convention makes the pairs of `AppointmentData` and `AppointmentProcessor` nice to read

Reply:
Good point. Done!